api/v2/sdks/sdk-ref-csharp-dotnet.md

```diff
@@ -45,53 +49,83 @@ The binary is published in this location:
 
 ## Initialization
 
-The initialization function configures the parameters necessary for the SDK to authenticate with the EUID service. It also allows you to configure retry intervals in the event of errors.
+DSPs should create an instance of the `BidstreamClient` class.
 
-| Parameter | Description | Recommended Value |
-| :--- | :--- | :--- |
-| `endpoint` | The endpoint for the EUID service. | N/A |
-| `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
 
-## Interface 
+| Parameter | Description |
+| :--- | :--- |
+| `endpoint` | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). | 
+| `authKey` | The API key. See [EUID Credentials](../getting-started/gs-credentials). |
+| `secretKey` | The client secret. See [EUID Credentials](../getting-started/gs-credentials). |
 
-The interface allows you to decrypt EUID advertising tokens and return the corresponding raw EUID. 
+## Interface
 
->NOTE: When you use an SDK, you do not need to store or manage decryption keys.
+The `BidstreamClient` class allows you to decrypt EUID tokens into raw EUIDs.
 
-If you're a DSP, for bidding, call the interface to decrypt an EUID advertising token and return the EUID. For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
+For details on the bidding logic for handling user opt-outs, see [DSP Integration Guide](../guides/dsp-guide.md).
 
-The following is the decrypt method in C#:
+Notes:
 
-```cs
-using UID2.Client.IUID2Client
- 
-var client = EUIDClientFactory.Create(_baseUrl, _authKey, _secretKey);
-client.Refresh(); //Note that Refresh() should be called once after create(), and then once per hour
-var result = client.Decrypt(_advertisingToken);
-```
+- The `SharingClient` class is part of the SDK but is not currently available for use in EUID.
 
-### Response Content
+- When you use an SDK, you do not need to store or manage decryption keys.
 
-Available information returned through the SDK is outlined in the following table.
+### Decryption Response Content
+
+When decrypting with the `SharingClient`, the SDK returns the following information:
```

Comment:
Maybe let's change this SharingClient to BidstreamClient for now as the rest of contents still applicable

Reply:
Oops did I remove the wrong one? Fixed now. Thx much.